src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +74,65 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+}
+
+class Block{
```

Comment:
These new classes should be its own Java class instead of being a nested class.
Additionally, there are missing header comments for the class. All non trivial classes/ methods should have javadoc format header comments explaining what the class is doing.